rest-service/manager_rest/rest/resources_v3_1/manager.py

```diff
@@ -142,7 +149,11 @@ def post(self):
             distro_release=_manager['distro_release'],
             fs_sync_node_id=_manager.get('fs_sync_node_id', '')
         )
-        return get_storage_manager().put(new_manager)
+        result = get_storage_manager().put(new_manager)
+        current_app.logger.info('Manager added successfully')
+        if _manager.get('fs_sync_node_id'):
+            add_manager()
```

Comment:
you know, if we just sent the list of managers here, to the mgmtworker, then the mgmtworker-side task wouldn't need to query for it, and so wouldn't need the rest-client or the admin-token

Reply:
I guess you're right.
Fixed